.pre-commit-config.yaml

```diff
@@ -0,0 +1,60 @@
+---
+default_stages: [pre-commit]
+
+default_install_hook_types: [pre-commit, commit-msg]
+
+ci:
+  autoupdate_commit_msg: 'pre-commit: autoupdate hooks'
+  autofix_prs: false
+  # docker is not supported on pre-commit.ci
+  skip: [shellcheck]
```

Comment:
@Al-an-21 CI breaks on this step, let's check whether it's needed here/on this repo

Reply:
This skip makes sense only if we use `shellcheck` later in the hooks. Here we do not use it, so it complains it does not know which hook to skip, since it is not defined.